src/app/collections/collection-sheet/collection-sheet.component.scss

```
@@ -6,19 +6,898 @@
+  ::ng-deep .mat-mdc-form-field-infix {
+    min-height: 40px !important;
+    padding-top: 8px !important;
+    padding-bottom: 8px !important;
+  }
+
+  ::ng-deep .mat-mdc-select-value-text {
+    font-size: 13px;
+    font-family: Manrope, sans-serif;
+  }
+
+  ::ng-deep .mdc-notched-outline > * {
```

Stylelint (17.3.0)

[error] Unexpected unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)